app/controllers/users_controller.rb

```diff
@@ -38,7 +38,7 @@ def update
   private
   
   def user_params
-    params.require(:user).permit(:name, :email)
+    params.require(:user).permit(:username, :email, :password)
```

Comment:
This is in your controller, but if you look in the stack trace, you can see that clearance is handling the create action in its controller. You can override / supercede and this should work, but you have to make it go through your controller.